nibabel/freesurfer/io.py

```diff
@@ -44,24 +45,52 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_geometry(filepath):
+def _read_volume_info(fobj):
+    volume_info = OrderedDict()
+    head = np.fromfile(fobj, '>i4', 3)
+    if any(head != [2, 0, 20]):
+        warnings.warn("Unknown extension code.")
+    else:
+        volume_info['head'] = head
+        for key in ['valid', 'filename', 'volume', 'voxelsize', 'xras', 'yras',
+                    'zras', 'cras']:
+            pair = fobj.readline().decode('utf-8').split('=')
+            if pair[0].strip() != key or len(pair) != 2:
+                raise IOError('Error parsing volume info.')
+            volume_info[pair[0]] = pair[1]
```

Comment:
Personal preference: I'd rather `volume_info` use keys without spaces, and then use `.ljust(6)` to get the old spacing back on write.

If I wanted to pull out xras, I'd find it more intuitive to do `volume_info['xras']`, not `volume_info['xras  ']`.

(Looks like I commented on the wrong diff earlier. Sorry.)